src/components/ReservationModal.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from 'styled-components';
+import { WHITE, BLACK } from '../colors';
```

Comment:
```suggestion
import { WHITE, BLACK } from '../utils/colors';
```
This file was moved in a recent commit, so it has to be updated.

Reply:
I committed the suggestion using GitHub's UI, the build should pass now :)